### PR TITLE
added tests to check continuity in magnetic field when subsequent sources share a face

### PR DESCRIPTION
**Pull request by kj5248**
## Linked Issues

Closes #{ID}

## Description

Additional tests for PolyhedralPrismCurrentSource.

One to test whether last and first face of subsequent sources in planar circuit matchup. Second is to test whether field matches when you compare two subsequent sources of half length with a single source of full length.

## Checklist

I confirm that I have completed the following checks:

- [ ] Tests run locally and pass `pytest tests --reactor`
- [ ] Code quality checks run locally and pass `pre-commit run --from-ref develop --to-ref HEAD`
- [ ] Documentation built locally and checked `sphinx-build -W documentation/source documentation/build`

### CODE REVIEW

**Comment by CoronelBuendia** [comment]
@kj5248 can you rebase this? Getting ready to merge #2703 following the agreement that it is OK that it is not totally general.

File: tests/magnetostatics/test_circuits.py [hunk 1]
```diff
@@ -294,6 +296,40 @@ def test_field_plot(self):
...
+        )
+        field2 = source2._rho * _field_fabbri(
+            source2._dcm[1],
+            np.array([source2._face_points[0]]),
+            np.array([source2._face_normals[0]]),
+            np.array([source2._mid_points[0]]),
+            np.array([3]),
+            point,
+        )
+        assert np.allclose(field1, -field2, rtol=0, atol=EPS)
```
Comment (CoronelBuendia) [inline tests/magnetostatics/test_circuits.py:330]: @kj5248 yeah... still don't know what to do about this.

Comment (kj5248) [reply tests/magnetostatics/test_circuits.py:330]: I think maybe just need to remove this part as its not true for this method? Especially as have the test that compares 2 half shapes to a full shape?

**Comment by CoronelBuendia** [comment]
closed as no longer required for #2703

**Pull request closed without merging**